packages/embark-utils/src/accountParser.js

```diff
@@ -7,13 +7,17 @@ import {getHexBalanceFromString} from './web3Utils';
 const {utils} = require('web3');
 
 const path = require('path');
+const ERROR_ACCOUNT = 'ERROR_ACCOUNT';
 
 class AccountParser {
   static parseAccountsConfig(accountsConfig, web3, dappPath, logger, nodeAccounts) {
     let accounts = [];
     if (accountsConfig && accountsConfig.length) {
       accountsConfig.forEach(accountConfig => {
         let account = AccountParser.getAccount(accountConfig, web3, dappPath, logger, nodeAccounts);
+        if (account === ERROR_ACCOUNT) {
+          process.exit(1);
+        }
```

Comment:
I don't think we should make use of `process` APIs here and rather have `parseAccountsConfig()` and `getAccount()` throw instead.

The caller show should have an opportunity to decide what to do with the error that's propagated.

Reply:
Agreed. I think exiting the process should generally be the responsibility of a component using the modules.